src/main/java/NetworkConfiguration.java

```diff
@@ -25,13 +25,11 @@ public NetworkConfiguration() {
                 .layer(0, new DenseLayer.Builder()
                         .nIn(NetworkProperties.INPUTS.property)
                         .nOut(NetworkProperties.HIDDEN_NODES.property)
-                        .activation(Activation.RELU)
```

Comment:
Should we have no activation function for layer 0 (and 1 for that matter)?

Reply:
We have the same activation function(SIGMOID) set for the whole network (line 20)